### PR TITLE
feature/pelagos 3508 add title blocks

### PR DESCRIPTION
**Pull request by snguyen1**

### CODE REVIEW

File: src/Pelagos/Bundle/AppBundle/Resources/views/DatasetSummary/dataset-summary.html.twig [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% extends "PelagosAppBundle:template:UI.html.twig" %}
-
+{% block title %}Dataset Summary{% endblock %}
```
Comment (fightingtexasaggie) [inline src/Pelagos/Bundle/AppBundle/Resources/views/DatasetSummary/dataset-summary.html.twig]: Lets add "and deletion tool" or similar

File: assets/css/pelagos-module.css [hunk 2]
```diff
@@ -79,3 +79,11 @@
...
     z-index: 1000;
 }
+
+.element-invisible {
+    position: absolute !important;
+    clip: rect(1px 1px 1px 1px);
+    clip: rect(1px, 1px, 1px, 1px);
+    overflow: hidden;
+    height: 1px;
+}
```
Comment (mickel1138) [inline assets/css/pelagos-module.css]: new line

File: assets/css/template.css [hunk 3]
```diff
@@ -78,6 +77,15 @@ h2 {
     margin-bottom: 1em;
 }
 
+label {
```
Comment (mickel1138) [inline assets/css/template.css]: Isn't this a little too ambiguous?

Comment (snguyen1) [reply assets/css/template.css]: labels in many pages were not the same as before

**Pull request merged**